### ISSUE TITLE
Sync tool checks are failing even if the repositories are clean

### ISSUE THREAD

**Issue by nathanaelhoun** [mattermost/mattermost-plugin-starter-template#153]
Hi! I am trying the relatively new synchronisation tool, but it always fails with this message :
```
failed check: "source" repository is not clean
exit status 1
```

Both of the source and target repositories are clean (`git status` outputs `nothing to commit, working tree clean`).

**Comment by hanzei** [comment]
Good catch. There seems to be an issue with the separate `go.mod` file in `/build`.
